src/screens/authentication.js

```
@@ -98,6 +98,16 @@ class AuthenticationScreen extends React.PureComponent<Props, State> {
     this.handleDetailsNavigation(AUTHENTICATION_TYPE.MAGIC_LINK);
   };
 
+  runMigrations = async () => {
+    try {
+      await migrationsRunner();
+      /* istanbul ignore next */
+      return this.hideSplashScreen();
+    } catch (error) {
+      return this.hideSplashScreen();
```

why do you hide the splash screen twice ? do it after the try catch

---

I don't hide it twice, and hide when the job's promise is fullfilled or rejected

---

like Vincent is there any difficulties to put `return this.hideSplashScreen();`  after the try catch ?

---

in the `catch` you could throw a simple error and handle it eslewhere